torch/lib/c10d/bin/test.sh

```diff
@@ -4,5 +4,5 @@ set -ex
 
 mkdir -p build
 cd build
-cmake ../
+cmake ../ -DCMAKE_INSTALL_PREFIX="$PWD/../../tmp_install"
```

Comment:
Haha that doesn't seem too nice

Reply:
Well.... this is where the current insta-build expects to find ATen and Gloo. Not ideal, but waiting for those to be independent targets. Until then I'm afraid we have to piggyback onto tools/build_pytorch_libs.sh and look for stuff in torch/lib/tmp_install.